src/app/static/src/tests/components/projects/projectHistory.test.ts

```diff
@@ -203,7 +207,7 @@ describe("Project history component", () => {
             }
         });
         const wrapper = getWrapper(testPropsData, mockStore);
-        const deleteLink = wrapper.find("#v-s11").findAll(".version-cell").at(3).find("a");
+        const deleteLink = wrapper.find("#v-s11").findAll(".version-cell").at(4).find("a");
```

Comment:
Maybe add a test to check that the version counter is updated when a version is deleted.

Reply:
Deleting the version doesn't actually happen within the component, as the component is reflecting the state which is being mocked here. We're checking that the delete action is invoked by clicking the link, and also separately checking that the version count is correct for the mocked state - that should be enough to unit test the component.